### ISSUE TITLE
Appium image strategy does not work with accessibility '~abc.png''

### ISSUE THREAD

**Issue by khoingo-lg** [webdriverio/webdriverio#5765]
My locator is {accessibility: '~abc.png'}.
It works well with the old strategy (webdriverio 5.23.0). After I upgrade the new version webdriverio 6.3.4, webdriverIO use image strategy for my locator. It does not work. It throws element not found.

My environment: 
Appium: 1.17.1
Webdriverio: 6.3.4
Device: iOS simulator 13.3

Could you have any advice for me?

_Originally posted by @khoingo-lg in https://github.com/webdriverio/webdriverio/pull/5633#issuecomment-679903560_

**Comment by christian-bromann** [comment]
Can you provide some logs from both WebdriverIO version?

**Comment by christian-bromann** [comment]
Closing due to inactivity.
